src/sdsc/xsl-checks/figures.xslc

```diff
@@ -170,6 +171,21 @@
       </xsl:call-template>
     </xsl:variable>
 
+    <!-- Empty fileref is bad. -->
+    <xsl:if test="string-length($fileref) &lt; 1">
+      <result type="error">
+        <xsl:call-template name="sourcehint"/>
+        <message>
+          No file name given for media object in
+          <xsl:call-template name="createid"/>
+          <xsl:text>.</xsl:text>
```

Comment:
Is the linebreak between `xsl:call-template` and `xsl:text` on purpose? Wouldn't the following code better reflect your intention?

```xml
<xsl:call-template name="createid"/><xsl:text>.</xsl:text>
```

or

```xml
<xsl:call-template name="createid"/>.
```